src/interfaces/ITokenURIGenerator.sol

```diff
@@ -1,9 +1,9 @@
 // SPDX-License-Identifier: BUSL 1.1
-pragma solidity 0.8.11;
+pragma solidity 0.8.16;
 
 import "./IOptionSettlementEngine.sol";
 
-/// @title A token URI geneartor for Claim NFTs
+/// @title A token URI generator for Claim NFTs
 /// @author 0xAlcibiades
```

Comment:
Update NatSpec authors to match [src/TokenURIGenerator.sol](https://github.com/valorem-labs-inc/valorem-core/pull/143/files#diff-25d286b3e5866a161b226fabd9b71e0002d8b28ec5b4e2215c929ce53e6eaca6)

Reply:
I'm just going to remove duplicate offer info from interfaces.